### PR TITLE
Bump * (transition status: failing to degraded)

### PR DESCRIPTION
**Pull request by benjaminapetersen**

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
```
check if the operator has not recreate removed console resources...
	console.go:70: changing console operator state to 'Managed'...
=== RUN   TestUnmanaged
--- FAIL: TestUnmanaged (6.35s)
	console.go:94: changing console operator state to 'Unmanaged'...
	framework.go:33: deleting console Deployment...
	framework.go:33: deleting console Service...
	framework.go:33: deleting console Route...
	framework.go:33: deleting console ConfigMap...
	unmanaged_test.go:20: waiting to check if the operator has not recreate deleted resources...
	unmanaged_test.go:29: deleted console Deployment was recreated
	console.go:70: changing console operator state to 'Managed'...
=== RUN   TestEditUnmanagedConfigMap
--- PASS: TestEditUnmanagedConfigMap (3.14s)
	console.go:94: changing console operator state to 'Unmanaged'...
	util.go:26: patching Data on the console ConfigMap
	util.go:33: polling for patched Data on the console ConfigMap
	console.go:70: changing console operator state to 'Managed'...
=== RUN   TestEditUnmanagedService
--- PASS: TestEditUnmanagedService (3.14s)
	console.go:94: changing console operator state to 'Unmanaged'...
	util.go:49: patching Annotation on the console Service
	util.go:56: polling for patched Annotation on the console Service
	console.go:70: changing console operator state to 'Managed'...
=== RUN   TestEditUnmanagedRoute
--- PASS: TestEditUnmanagedRoute (3.15s)
	console.go:94: changing console operator state to 'Unmanaged'...
	util.go:72: patching TargetPort on the console Route
	util.go:79: polling for patched TargetPort on the console Route
	console.go:70: changing console operator state to 'Managed'...
FAIL
FAIL	github.com/openshift/console-operator/test/e2e	36.896s
[ERROR] PID 206: hack/test-e2e.sh:5: `KUBERNETES_CONFIG=${KUBECONFIG} GOCACHE=off go test -timeout 30m -v ./test/e2e/` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/test-e2e.sh:5: `KUBERNETES_CONFIG=${KUBECONFIG} GOCACHE=off go test -timeout 30m -v ./test/e2e/`
[INFO]   Exiting with code 1.
make: *** [test-e2e] Error 1
```
@jhadvig I'm always puzzled when we see these fail from a dependency bump.

**Comment by benjaminapetersen** [comment]
Looking into that issue.

File: test/e2e/unmanaged_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func TestUnmanaged(t *testing.T) {
 	checkErr := <-errChan
 
 	if checkErr != nil {
-		t.Fatal(checkErr)
+		t.Fatalf("error: %s", checkErr)
```
Comment (benjaminapetersen) [inline test/e2e/unmanaged_test.go]: This is what the rest of the tests do in this file.

File: pkg/console/operator/operator.go [hunk 2]
```diff
@@ -178,10 +178,10 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 
 	// all configs needed to do a sync
 	if err := c.handleSync(operatorConfig, consoleConfig, infrastructureConfig); err != nil {
-		c.SyncStatus(c.ConditionFailing(operatorConfig, "SyncLoopError", "Operator sync loop failed to complete."))
+		c.SyncStatus(c.ConditionDegraded(operatorConfig, "SyncLoopError", "Operator sync loop failed to complete."))
```
Comment (benjaminapetersen) [inline pkg/console/operator/operator.go]: Just renames.

**Comment by spadgett** [comment]
```
--- FAIL: TestEditUnmanagedConfigMap (2.12s)
	console.go:94: changing console operator state to 'Unmanaged'...
	util.go:26: patching Data on the console ConfigMap
	unmanaged_test.go:40: error: configmaps "console-config" not found
```

**Comment by benjaminapetersen** [comment]
Wonder if that is a timout limit issue.

**Comment by benjaminapetersen** [comment]
/retest

I'm curious

**Comment by benjaminapetersen** [comment]
```
unmanaged_test.go:40: error: configmaps "console-config" not found
```

**Comment by benjaminapetersen** [comment]
Checking to see if this resolves with an increased timeout.

**Comment by benjaminapetersen** [comment]
Yup, just a timeout issue.

**Comment by benjaminapetersen** [comment]
Eliminated the iteration logging, but left the longer timeout.

**Comment by benjaminapetersen** [comment]
🐹

**Pull request merged**